### ISSUE TITLE
4k Firestick Accessibility Commands Solutions not Works

### ISSUE THREAD

**Issue by MPenSol** [BaronKiko/LauncherHijack#72]
I’m sorry to open the thread, today I’m done the steps to install the own launcher but it don’t work’s. It seems Amazon has change something since the OS Version 6.2.7.3.

The Following steps are working fine it the past, but now i configure a new 4k Stick with the Fire OS Version 6.2.7.3 and it it doesn't works:

_Ok, here is a quick and dirty solution:
Edit: before doing this first uninstall LauncherHijack if you have it still installed.
First install the cloned LauncherHijack apk from here
Then install the apk inside this zip
(don't pay attention to its UI if it shows, it's just a template, the important part is that when it opens it configures the accessibility variables as needed)
Then grant it the required permission by running:
adb shell pm grant com.example.settest.settingstest android.permission.WRITE_SECURE_SETTINGS_

__
Have anyone a new solution?

**Comment by dashriprock007** [comment]
Yes, odd. I have one 4k unit that my launcher hijack is still working on, and the other seems broken even after I tried reinstalling like above...

**Comment by oathzed** [comment]
Its deeper then giving something accessibility permission. I was able to give button mapper accessibility, but when i tried to add a keymap, nothing registered. So Amazon really changed something.
